MoltenVK/MoltenVK/API/mvk_private_api.h

```diff
@@ -240,6 +240,8 @@ typedef struct {
 	MVKConfigCompressionAlgorithm shaderSourceCompressionAlgorithm;            /**< MVK_CONFIG_SHADER_COMPRESSION_ALGORITHM */
 	VkBool32 shouldMaximizeConcurrentCompilation;                              /**< MVK_CONFIG_SHOULD_MAXIMIZE_CONCURRENT_COMPILATION */
 	float timestampPeriodLowPassAlpha;                                         /**< MVK_CONFIG_TIMESTAMP_PERIOD_LOWPASS_ALPHA */
+	VkBool32 useMetalPrivateAPI;                                               /**< MVK_CONFIG_USE_METAL_PRIVATE_API */
+	uint32_t _reserved;
```

Comment:
The compiler insisted on making the struct size a multiple of 8, so the `static_assert()` in `MoltenVK/MoltenVK/Utility/MVKEnvironment.cpp` fails unless this is present in `MVKConfigMembers.def`, and therefore it must be present here as well.

Reply:
Ah. So I just got lucky with struct boundaries when I added that assertion test!

That assertion is just a dev safety to ensure we keep `MVKConfigMembers.def` in sync with the definition of `MVKConfiguration`.

Darn. If `MVKConfiguration` wasn't part of the public API, we could just let its structure be defined by `MVKConfigMembers.def`. Maybe we should anyway, since this is really a private API now that `VK_EXT_layer_settings` is active.

But that's a larger discussion for another time.

For now, I'm imagining another dev coming by months down the road, and thinking that member is reserved by someone, and adding another config member on after it. So...to clarify intended usage, maybe change it from `uint32_t _reserved` to something like `uint32_t _unused_struct_padding`, so its purpose is clearer? And maybe add a comment indicating it's purpose.